backup_and_restore/backing-up-etcd.adoc

```diff
@@ -0,0 +1,30 @@
+[id="backup-etcd"]
+= Backing up etcd
+include::modules/common-attributes.adoc[]
+:context: backup-etcd
+
+toc::[]
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
+
+It is recommended to back up your cluster's etcd data regularly and store in a
+secure location ideally outside the {product-title} environment. Do not take an
+etcd backup before the first certificate rotation completes, which occurs 24
+hours after installation, otherwise the backup will contain expired
+certificates. It is also recommended to take etcd backups during non-peak usage
+hours, as it is a blocking action.
+
+Once you have an etcd backup, you can xref:../backup_and_restore/disaster_recovery/scenario-1-infra-recovery.adoc#dr-infrastructure-recovery[recover from lost master hosts]
+and xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore back to a previous cluster state].
+
+// TODO: Removed "and where the etcdctl tool is installed." - okay, or is that
+// necessary??
+// Does it have to be a master host?
+You can perform the xref:../backup_and_restore/backing-up-etcd.adoc#backing-up-etcd-data_backup-etcd[etcd data backup process]
```

Comment:
@hexfusion Can you also check this whole section, it's new. I reused some wording from the 3.x docs, but had a question on this part. It said that you had to have etcdctl installed (but I removed that because I thought your script installed if needed, can you confirm?) and also wanted to check whether this "any host that has connectivity to the etcd cluster" had to be a master host or not.